flang/include/flang/Lower/Image.h

```diff
@@ -0,0 +1,83 @@
+//===-- Lower/Image.h -- image related lowering -----------------*- C++ -*-===//
```

Comment:
Would Coarray.h/.cpp fit for these files ? Image.h is fine but Coarray.h may be help to grasp faster what this is about.

Reply:
I can rename them, but I thought "all things image related" was more general than simply "coarray".